### PR TITLE
Adds response for item 16.

### PR DESCRIPTION
**Pull request by chenhyhy**

### CODE REVIEW

File: report.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@ What makes people happy?
 13.
 14.
 15.
-16.
+16. holiday, food  ^-^
```
Comment (lungnagac-spg) [inline report.txt:20]:
```suggestion
16. holiday, my foot  ^-^
```

**Review by lungnagac-spg — CHANGES_REQUESTED** [review]
Fix THE TYPO !!!

File: report.txt [hunk 2]
```diff
@@ -17,7 +17,7 @@ What makes people happy?
 13.
 14.
 15.
-16.
```
Comment (soonsengtoh) [inline report.txt:20]: not good enough

**Review by soonsengtoh — CHANGES_REQUESTED** [review]
na

**Review by soonsengtoh — APPROVED** [review]

**Review by lungnagac-spg — APPROVED** [review]

**Pull request merged**